resources/web/style/on_this_page.pcss

```diff
@@ -5,16 +5,19 @@
 
   .heading-level-1 {
     display: block;
-    padding-left: 1.5em !important;
+    padding-left: 1em !important;
+    a {
+      font-size: 0.9em;
+    }
```

Comment:
Should we remove this? Only `.heading-level-1` has a unique font-size—`*0`, `*2`, and `*3` do not. The OTP looks a little goofy when there are multiple heading levels. I'm not sure why I added this in the first place 🙃.

Reply:
```suggestion
```